README.md

```diff
@@ -2,239 +2,293 @@
 
 ![MPS SplashScreen](Documentation/Media/splashscreen.png)
 
-A third-person multiplayer game sample for O3DE, where robots battle one another for dominance in a multi-tiered, under-construction, Starbase
+The MultiplayerSample Project is a third-person multiplayer game built on Open 3D Engine (O3DE), where robots battle one another for dominance in a multi-tiered starbase.
```

Comment:
nit: "under-construction" i believe was meant to describe the condition of the starbase (it's a construction zone), not that the project itself is under construction, so i think it's still a valid descriptive phrase. Maybe "multi-tiered partially-constructed starbase" or "multi-tiered starbase construction site"?

Reply:
Added "under construction"